postcss.config.js

```diff
@@ -1,13 +1,24 @@
+const cssnano = require("cssnano")({preset: "default"})
+let minimize = false
+
+const plugins = [
+  require("postcss-import"),
+  require("tailwindcss")("./tailwind.config.js"),
+  require("postcss-flexbugs-fixes"),
+  require("postcss-preset-env")({
+    autoprefixer: {
+      flexbox: "no-2009"
+    },
+    stage: 3
+  })
+]
+
+if (process.env.NODE_ENV === 'production') {
+  plugins.push(cssnano)
```

Comment:
Just a question: What does cssnano do and how is it different from the minimize option?

Reply:
It's used in conjunction with the minimize option.  but i have to walk this back because of these potential issues: https://alexpeattie.com/blog/psa-dont-use-cssnano-with-postcss/ which links to this comment on the gh issue: https://github.com/cssnano/cssnano/issues/659#issuecomment-444025691.

also, doing it this way automatically considers the production environment, so we won't need to do an explicit check